cypress/integration/run-processes/run-export-d2cc-process-business-date-view-acceptance-test.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import * as gc from "../../support/gridcapa-util";
+import * as minio from "../../support/minio";
+import * as ftp from "../../support/ftp-browser.js";
+
+
+const minioUser = Cypress.env('GRIDCAPA_MINIO_USER');
+const minioPassword = Cypress.env('GRIDCAPA_MINIO_PASSWORD')
+const URL = '/cse/export/d2cc'
+const DATE = '2022-01-28'
+const TIME = '16:30'
+const TIMEOUT = 60000;
+const TIMEOUT_UPLOAD = 300000;
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
why?

Reply:
I just copy these lines from run-export-idcc-process-acceptance-test.js which is my base for this test